packages/theme/src/theme.ts

```diff
@@ -322,4 +322,4 @@ export const theme: ConfigBase['theme'] = {
       },
     },
   },
-};
+} satisfies ConfigBase['theme'];
```

Comment:
I'm using a `satisfies` here so that they can import the `theme` and then override values and have autocompletion.